lib/common/utils/router.dart

```diff
@@ -26,6 +26,8 @@ class FlutterConRouter {
 
   static final GlobalKey<NavigatorState> _globalNavigatorKey =
       GlobalKey<NavigatorState>();
+  static GlobalKey<NavigatorState> get globalNavigatorKey =>
+      _globalNavigatorKey;
```

Comment:
Why have 2 when instead you can convert the above to a public one that you can use where you need it to be used.